src/engine/NamedResultCache.cpp

```diff
@@ -6,18 +6,28 @@
 
 #include "engine/NamedResultCache.h"
 
+#include <algorithm>
+
+#include "engine/NamedResultCacheSerializer.h"
+#include "util/Serializer/FileSerializer.h"
+
 // _____________________________________________________________________________
 std::shared_ptr<ExplicitIdTableOperation> NamedResultCache::getOperation(
     const Key& name, QueryExecutionContext* qec) {
   const auto& result = get(name);
-  const auto& [table, map, sortedOn, localVocab, cacheKey, geoIndex] = *result;
+  const auto& [table, map, sortedOn, localVocab, cacheKey, geoIndex, allocator,
+               blankNodeManager] = *result;
   auto resultAsOperation = std::make_shared<ExplicitIdTableOperation>(
       qec, table, map, sortedOn, localVocab.clone(), cacheKey);
   return resultAsOperation;
 }
 
 // _____________________________________________________________________________
-auto NamedResultCache::get(const Key& name) -> std::shared_ptr<const Value> {
+auto NamedResultCache::get(const Key& name) const
+    -> std::shared_ptr<const Value> {
+  // Note the `wlock` on a `mutable` cache inside a `const` member function here
+  // is benign as we don't modify the contents of the cache (only the LRU
```

Comment:
I don't get how a `wlock` can be benign (same for the opposite). What does this comment mean?